app/controllers/users.py

```diff
@@ -429,8 +429,8 @@ def patch(self, user_id):
         """
         """
         try:
-            user_schema = UserSchema(only=("name", "is_public"))
-
+            
+            user_schema = UserSchema(partial=True)
```

Comment:
Thanks for this Chris . Though this poses some security concerns on the endpoint as this means someone can modify fields such as `is_beta_user` , `verified `  once they can bundle them with the payload which are only to be updated under certain circumstances . 

```suggestion
            user_schema = UserSchema(only=("name", "is_public" , "organisation"))
```